build-system/tasks/helpers.js

```diff
@@ -294,11 +313,31 @@ function compileMinifiedJs(srcDir, srcFilename, destDir, options) {
         }
       })
       .then(() => {
-        let name = options.minifiedName;
+        let name = minifiedName;
         if (options.latestName) {
           name = `${name} → ${options.latestName}`;
```

Comment:
nit: not a huge fan of this syntax, I prefer ```name += ` → ${options.latestName}`;``` (here and below) which I feel is more readable, but that's up to you

Reply:
Fixed.